packages/amplication-server/src/core/build/build.module.ts

```diff
@@ -35,9 +37,15 @@ import { CommitModule } from '../commit/commit.module'; // eslint-disable-line i
     forwardRef(() => ResourceModule),
     ServiceSettingsModule,
     QueueModule,
-    forwardRef(() => CommitModule)
+    forwardRef(() => CommitModule),
```

Comment:
Do we need `forwardRef()` here? Can you try to use `CommitModule` without `forwardRef()`

Reply:
Not sure, but it's out of scope of this PR. Let's check it separately.